README.md

```diff
@@ -1 +1,97 @@
-# TDEI-python-gtfs-pathways-validation
\ No newline at end of file
+# TDEI-python-gtfs-pathways-validation
+
+## Introduction 
+Service to Validate the GTFS pathways file that is uploaded. At the moment, the service does the following:
+- Listens to the topic _gtfs-pathways-validation_ for any new message (that is triggered when a file is uploaded)
```

Comment:
This is no longer true, right? Please update the readme file to reflect the code.